api/src/stampy_chat/chat.py

```diff
@@ -141,7 +141,30 @@ def construct_prompt(query: str, mode: str, history: List[Dict[str, str]], conte
 import time
 import json
 
-def talk_to_robot_internal(index, query: str, mode: str, history: List[Dict[str, str]], k: int = STANDARD_K, log: Callable = print):
+
+def check_openai_moderation(prompt: Prompt, query: str, log: Callable):
+    # 3. Run both the standalone query and the full prompt through
+    # moderation to see if it will be accepted by OpenAI's api
```

Comment:
duplicate comment doesn't make sense when factoring this out of the function, probably accidental.
```suggestion
```